File: FriendlyDonations/app/src/main/java/br/com/friendlydonations/application/login/LoginActivity.java
package br.com.friendlydonations.application.login;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

import br.com.friendlydonations.R;
import br.com.friendlydonations.application.main.MainActivity;
import br.com.friendlydonations.application.web.WebLoadActivity;
import br.com.friendlydonations.shared.BaseActivity;
import butterknife.ButterKnife;
import butterknife.OnClick;

import static br.com.friendlydonations.application.web.WebLoadActivity.TITLE_EXTRA;
import static br.com.friendlydonations.application.web.WebLoadActivity.URL_PATH_EXTRA;
import static br.com.friendlydonations.application.web.WebLoadActivity.WebUrlPath.TERMS;

/**
 * Created by brunogabriel on 24/04/17.
 */

public class LoginActivity extends BaseActivity implements LoginView {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.content_login);
        ButterKnife.bind(this);
        startActivity(new Intent(this, MainActivity.class));
    }

    @OnClick(R.id.about_button)
    protected void onClickAboutButton() {
        startActivity(new Intent(this, WebLoadActivity.class)
                .putExtra(URL_PATH_EXTRA, TERMS.getValue())
                .putExtra(TITLE_EXTRA, getString(R.string.login_about_and_use_terms)));
    }
}

File: FreeStuffs-Android/app/src/main/java/br/com/freestuffs/shared/login/LoginActivity.kt
package br.com.freestuffs.shared.login

import android.os.Bundle
import br.com.freestuffs.R
import br.com.freestuffs.shared.application.BaseActivity

/**
 * Created by bruno on 24/08/2017.
 */
class LoginActivity: BaseActivity(), LoginContract.View {
    override var presenter: LoginContract.Presenter?=null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        presenter = LoginPresenter(this)
    }

    override fun startView() {
        // TODO: Setup view
    }
}